api/about.json

```diff
@@ -1,9 +1,9 @@
 {
  "0.0.1": [
    {
-    "logo": "http://bit.ly/2niBg5Y",
-    "website":"https://pydelhi.org/",
-    "details":"PyDelhi is a volunteer organization, group of technically inclined students, professionals, teachers and entrepreneurs who want to learn something new having a common primary interest in Python programming language." 
+    "logo": "https://paste.opensuse.org/89323686",
```

Comment:
@Warlord77 It should point to raw file like: https://paste.opensuse.org/view/raw/89323686